source/agora/consensus/EnrollmentManager.d

```diff
@@ -457,9 +457,10 @@ public class EnrollmentManager
         const start_height = enrolled_height + 1;
```

Comment:
> Sometimes, the requested index of pre-image is above the cycle minus 1

Can you add a small note when this would actually happen?

Also there is a typo: munus => minus

Reply:
> Can you add a small note when this would actually happen?
> Also there is a typo: munus => minus

I added the note to commit message and fix the typo.

Reply:
> Can you add a small note when this would actually happen?
> Also there is a typo: munus => minus

I added the note to commit message and fix the typo.